examples/01-filter/glyphs.py

```diff
@@ -50,7 +50,7 @@
 ).T
 
 # add and scale
-sphere.vectors = vectors * 0.3
+sphere["vector"] = vectors * 0.3
```

Comment:
This should probaby stay `vectors` for `glyph` to catch them, right?
```suggestion
sphere["vectors"] = vectors * 0.3
```

Reply:
I think it is best to align with "vectors" since it used above in this example for another case.  However, this was a good catch as pv.Sphere includes "Normals" which is set as the active vector.  We need to override this now.  This is addressed in afba2eedeb8bcbd74b64868ef50525a0e3969023.